### ISSUE TITLE
Using spans

### ISSUE THREAD

**Issue by AntonC9018** [ArjunVachhani/XlsxHelper#7]
If you never copied from raw bytes to string objects, it would probably be even faster. I'm actually really surprised the standard .NET API's for XML return strings, rather than just pointing at bytes in the buffer. It would be unsafe, but it's really important for high-performance apps.

**Comment by ArjunVachhani** [comment]
@AntonC9018 agreed, minimizing time spent in GC can increase the performance. As you said .NET framework does not support `Span<T>` based API for `XmlReader`.

**Comment by ArjunVachhani** [comment]
Closing as XmlReader does not support Span<T>
